tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -9,6 +10,21 @@ const taskFilterSelector = '.card.tasks .table-filter a';
 const taskFilter = () => $(taskFilterSelector);
 const taskFilters = () => $$(taskFilterSelector);
 const contactList = () => $('#contacts-list');
+const newPlaceName = () => $('[name="/data/init/custom_place_name"]');
```

Comment:
I'm wondering if these elements need to go into a config specific page object. I don't know the html enough to say it's different between one config to the other. What are your thoughts?

Reply:
> I'm wondering if these elements need to go into a config specific page object. I don't know the html enough to say it's different between one config to the other. What are your thoughts?

I think these come from forms and they look same for both [default](https://github.com/medic/cht-core/blob/master/config/default/forms/contact/clinic-create.xml#L127) and [standrad](https://github.com/medic/cht-core/blob/master/config/standard/forms/contact/clinic-create.xml#L152)